dockerunit-consul/pom.xml

```diff
@@ -43,6 +44,12 @@
       <artifactId>vertx-core</artifactId>
       <version>${vertx.core.version}</version>
     </dependency>
+    <dependency>
+      <groupId>org.projectlombok</groupId>
+      <artifactId>lombok</artifactId>
+      <version>${lombok.version}</version>
+      <scope>provided</scope>
```

Comment:
I'm wondering why the scope here is `provided`?

Reply:
I have uploaded a new snapshot.
You can try out whether it works as expected on Mac by pulling and executing the usual
`./mvnw test -P container-tests -Ddocker.host=localhost -Ddocker.bridge.ip=172.17.0.1`
from the spring-boot-example.
Once you confirm I'll merge and release 1.0.0-M2 to Central.

Reply:
Just tested it. It works in my machine 😃

Reply:
Thanks Adrian :-) Releasing now